docker-compose.yml

```diff
@@ -16,17 +16,16 @@ services:
     environment:
       # The Teamscale memory limit (alternative to storing it in jvm.properties)
       TEAMSCALE_MEMORY: 4G
-      # Overwrite settings from config/teamscale.properties
+      # You can specify Teamscale properties here
+      # teamscale.properties is not considered at all when this environment variable is set
       TS_PROPERTIES: |-
+        database.type=leveldb
         engine.workers=2
         instance.name=v7.7
-      # Overwrite JVM options
-      TEAMSCALE_OPTS: >-
-        -Dcom.teamscale.mergerequest.disable-automatic-webhook-creation=true
-        -Dcom.teamscale.enable-monitoring-uploads=true
-        -Dcom.teamscale.commit-tree-timestamp-assertion=true
-        -XX:HeapDumpPath=/var/teamscale/logs
-        -XX:+HeapDumpOnOutOfMemoryError
+      # You can also overwrite JVM options if necessary
+      # jvm.properties is not considered at all if this environment variable is set
```

Comment:
not true , see above. The file has the cryptic JVM_EXTRA_ARGS which is still valid

Reply:
this is super confusing :P I'll adjust the text to try and make this clear